cs/build/nuget/Common.targets

```diff
@@ -34,22 +34,44 @@
     </BondImportDirectory>
   </ItemDefinitionGroup>
 
+  <!-- Let Visual Studio know about the BondCodegen item so it can display
+       it in the UI as a potential action for files. -->
   <ItemGroup>
     <AvailableItemName Include="BondCodegen" />
   </ItemGroup>
 
-  <!--
-    * BondCodegenCs - transform .bond files into .cs files
-    *    If we use output transforms from BondCodegen to the resulting targets we can get partial/minimal rebuilds
-    *    on change, but this doesn't reflect the internal import logic of bond.  A change to common.bond would
-    *    only rebuild common*cs, not all the others that import common.bond.  Without a bond parser prepass we
-    *    must rebuild all .bond files when any of them change.  We force that by putting a non-transform output
-    *    (bondcodegen.done) in the list so IFF any input changes, then all inputs rebuild since MSBuild can't
-    *    know what should create the unmapped output item and so can't be selective.
-    *    NOTE - this still won't catch changes to files outside declared BondCodegen elements.
-  -->
+  <!-- Attempt to find gbc in a platform-agnostic way. -->
+  <PropertyGroup>
+    <_BondExe>gbc</_BondExe>
+    <_BondExe Condition="Exists('$(BOND_COMPILER_PATH)\gbc.exe')">$(BOND_COMPILER_PATH)\gbc.exe</_BondExe>
+    <_BondExe Condition="Exists('$(BOND_COMPILER_PATH)\gbc')">$(BOND_COMPILER_PATH)\gbc</_BondExe>
```

Comment:
It would use the second one. Properties are evaluated top-to-bottom, and the last one that evaluates wins.